velox/benchmarks/QueryBenchmarkBase.cpp

```diff
@@ -333,13 +337,18 @@ QueryBenchmarkBase::run(const TpchPlan& tpchPlan) {
         if (!taskCursor->noMoreSplits()) {
           for (const auto& entry : tpchPlan.dataFiles) {
             for (const auto& path : entry.second) {
-              auto splits = facebook::velox::cudf_velox::cudfIsRegistered() &&
-                      facebook::velox::connector::getAllConnectors().count(
-                          cudf_velox::exec::test::kParquetConnectorId) > 0 &&
-                      facebook::velox::cudf_velox::cudfTableScanEnabled()
-                  ? listCudfSplits(
-                        path, 1 /* numSplitsPerFile = 1 for cudf */, tpchPlan)
-                  : listSplits(path, numSplitsPerFile, tpchPlan);
+              auto splits = [&]() {
+                // Check if we need cuDF splits
+                if (cudf_velox::cudfIsRegistered() &&
+                    connector::getAllConnectors().count(
+                        cudf_velox::exec::test::kParquetConnectorId) > 0 &&
+                    cudf_velox::cudfTableScanEnabled()) {
```

Comment:
Hi @karthikeyann @devavret, I need a suggestion:

Is it worth removing `listCudfSplits` altogether and modify `listSplits` to create `HiveConnectorSplits` here as it does now but with `connectorId = kParquetConnectorId`, `numSplitsPerFile=1`, and `fileFormat == PARQUET`, if this logic at L342-L354 is true?

Reply:
Although, another suggestion required:

Should I instead open this PR in https://github.com/facebookincubator/velox?